### PR TITLE
Add propTypes to Button for colorchange

### PR DESCRIPTION
**Pull request by Phil9219**

### CODE REVIEW

**Comment by Sina-Sche** [comment]
lgtm 👍

File: client/src/components/Input/Input.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ const Input = styled.input`
   background: none;
   color: var(--secondary-color);
   font-family: "Heiti SC", adobe-heiti-std, sans-serif;
-  font-size: 14px;
+  font-size: 15px;
```
Comment (SvLowman) [inline client/src/components/Input/Input.js:10]: You could use 'rem' instead of 'px'.

File: client/src/components/Button/Button.js [hunk 2]
```diff
@@ -1,15 +1,26 @@
...
+    props.main ? "var(--secondary-color)" : "var(--primary-color)"};
+
+  color: ${(props) =>
+    props.main ? "var(--primary-color)" : "var(--secondary-color)"};
+
   border: 2px solid var(--secondary-color);
   cursor: pointer;
-  color: var(--secondary-color);
+
   font-size: 29px;
```
Comment (SvLowman) [inline client/src/components/Button/Button.js:17]: You could use 'rem' instead of 'px'.

**Review by SvLowman — APPROVED** [review]
Looks good!

**Review by Naasgul — APPROVED** [review]
lgtm! I prefer using rem instead of px, but that's your choice 👍

File: client/src/components/Input/Input.js (hunk 1 above)
Comment (Phil9219) [reply client/src/components/Input/Input.js:10]: Will do. Thanks

File: client/src/components/Button/Button.js (hunk 2 above)
Comment (Phil9219) [reply client/src/components/Button/Button.js:17]: THX

**Pull request merged**